### PR TITLE
ci(docker): initial version of docker build and push tags

### PR DESCRIPTION
**Pull request by mmpetarpeshev**
We have to decide - Do we want PR builds against  main branch or only against develop.To me looks like unnecessary duplication in most of the cases , but maybe could be useful if there is something like hot fixes(if they goes through PR's).

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Publish Docker Superhero Wallet image
+
+on:
+  push:
+    branches: [ master, develop ]
```
Comment (kenodressel) [inline .github/workflows/docker.yml:5]: here i guess we could only check for master

Comment (mmpetarpeshev) [reply .github/workflows/docker.yml]: Done

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  main:
+    name: Wallet Build/Test/Docker
```
Comment (kenodressel) [inline .github/workflows/docker.yml:13]: I guess its only docker right?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -1,77 +0,0 @@
-name: Test, build, and deploy
```
Comment (kenodressel) [inline .github/workflows/main.yml:1]: I dont think we can remove this quite yet, there is a lot more in here

Comment (mmpetarpeshev) [reply .github/workflows/main.yml:1]: We are not going to merge , before everything is done - like staging deployments.

File: .github/workflows/docker.yml (hunk 2 above)
Comment (mmpetarpeshev) [reply .github/workflows/docker.yml:13]: Depends on how we look on it , inside the docker we have buid, test and we have docker build and push

**Comment by kenodressel** [comment]
Why did we delete the issue templates?

**Pull request closed without merging**